keras_ocr/tools.py

```diff
@@ -33,6 +33,8 @@ def read(filepath_or_buffer: typing.Union[str, io.BytesIO]):
         assert os.path.isfile(filepath_or_buffer), \
             'Could not find image at path: ' + filepath_or_buffer
         image = cv2.imread(filepath_or_buffer)
+    else:
+        raise Exception("Wrong input file type: should be 'str' or 'io.BytesIO' ")
```

Comment:
Thank you so much for raising this! I agree we need to handle this differently so uses have a better idea of what went wrong.

Could we change this message to be a little broader? It could be the user provided a valid filepath but OpenCV failed to read it (e.g., if it's a format that OpenCV doesn't support). Perhaps this message could read, "Failed to read image."

If you could make a change to that effect, I'd be happy to get it merged in.